src/bin/miri.rs

```diff
@@ -63,6 +64,7 @@ struct MiriCompilerCalls {
     many_seeds: Option<ManySeedsConfig>,
     /// Settings for using GenMC with Miri.
     genmc_config: Option<GenmcConfig>,
+    tracing_guard: Option<TracingGuard>,
```

Comment:
Why do we need a new field here and what does this do?

Reply:
I added a doc comment